docs/CONAN_USER_DOCUMENTATION.md

```diff
@@ -1,3 +1,14 @@
-# Documentation
+# How to use Conan with NXRM
 
-Coming soon!
+1. Install Conan (if on OSX you can do this via ```brew install conan```)
+2. Create the conan repositories in NXRM (e.g. conan-hosted)
```

Comment:
Is it worth step 2 being install the plugin for people who don't read thoroughly and just skipped/were directed here?

Reply:
I don't want to duplicate so added a note 
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/28/commits/e325b0119a60424c5c84e4b3e9331431b3135dc6

Reply:
Fair, thanks.
I was thinking "2. Install plugin per <link>" but that also works!